content/tutorials/integration-tutorials/monai_3d_segmentation.md

```diff
@@ -581,7 +581,7 @@ for epoch in epoch_progress_bar:
 artifact.wait()
 ```
 
-Instrumenting the code with `wandb.log` not only enables tracking all metrics associated with the training and validation process, but also the all system metrics (our CPU and GPU in this case) on the W&B dashboard.
+Instrumenting the code with `wandb.log` not only enables tracking all metrics associated with the training and validation process, but also logs all system metrics (our CPU and GPU in this case) on the W&B dashboard.
```

Comment:
Just a heads up that these docs are autogenerated (manual and have not been updated in ages) so if we were to regenerate these, this change will be overwritten.